fdbserver/ptxn/test/TestTLogServer.actor.cpp

```diff
@@ -789,29 +792,50 @@ TEST_CASE("/fdbserver/ptxn/test/pop_data") {
 	state ptxn::test::TestDriverOptions options(params);
 	state std::vector<Future<Void>> actors;
 	(const_cast<ServerKnobs*> SERVER_KNOBS)->BUGGIFY_TLOG_STORAGE_MIN_UPDATE_INTERVAL = 0.5;
+	(const_cast<ServerKnobs*> SERVER_KNOBS)->TLOG_SPILL_THRESHOLD = 0;
+
 	state std::shared_ptr<ptxn::test::TestDriverContext> pContext = ptxn::test::initTestDriverContext(options);
 
 	for (const auto& group : pContext->tLogGroups) {
 		ptxn::test::print::print(group);
 	}
 	state ptxn::StorageTeamID storageTeamID = pContext->storageTeamIDs[0];
-
 	state std::string folder = "simfdb/" + deterministicRandom()->randomAlphaNumeric(10);
 	platform::createDirectory(folder);
 
-	wait(startTLogServers(&actors, pContext, folder, true));
+	wait(startTLogServers(&actors, pContext, folder));
 	state IKeyValueStore* d = pContext->kvStores[pContext->storageTeamIDTLogGroupIDMapper[storageTeamID]];
+	state IDiskQueue* q = pContext->diskQueues[pContext->storageTeamIDTLogGroupIDMapper[storageTeamID]];
 
 	state std::pair<std::vector<Standalone<StringRef>>, std::vector<Version>> res =
 	    wait(commitInjectReturnVersions(pContext, storageTeamID, pContext->numCommits));
 	state std::vector<Standalone<StringRef>> expectedMessages = res.first;
-	wait(verifyPeek(pContext, storageTeamID, pContext->numCommits));
+	// wait(verifyPeek(pContext, storageTeamID, pContext->numCommits));
```

Comment:
Can you add a TODO here--when to reenable this?

Reply:
will do, can have it after peek works for on-disk data, I think I had it, but somehow lost it..

Reply:
oh i have comments in other places, miss this one